Makefile

```diff
@@ -95,64 +107,84 @@ test-unit-go:
 # Build / Publish                                                              #
 ################################################################################
 
-.PHONY: build
-build: build-images xbuild-cli
-
-.PHONY: build-images
-build-images: build-apiserver build-scheduler build-observer
-
 .PHONY: build-logger-linux
 build-logger-linux:
-	docker build \
-		-t $(DOCKER_IMAGE_PREFIX)brigade-logger-linux:$(IMMUTABLE_DOCKER_TAG) \
-		- < v2/logger/Dockerfile.linux
-	docker tag $(DOCKER_IMAGE_PREFIX)brigade-logger-linux:$(IMMUTABLE_DOCKER_TAG) $(DOCKER_IMAGE_PREFIX)brigade-logger-linux:$(MUTABLE_DOCKER_TAG)
+	$(KANIKO_DOCKER_CMD) /kaniko/executor \
+		--dockerfile /workspaces/brigade/v2/logger/Dockerfile.linux \
+		--context dir:///workspaces/brigade/logger \
+		--no-push
 
 .PHONY: build-%
 build-%:
-	docker build \
-		-f v2/$*/Dockerfile \
-		-t $(DOCKER_IMAGE_PREFIX)brigade-$*:$(IMMUTABLE_DOCKER_TAG) \
-		--build-arg VERSION='$(VERSION)' \
-		--build-arg COMMIT='$(GIT_VERSION)' \
-		.
-	docker tag $(DOCKER_IMAGE_PREFIX)brigade-$*:$(IMMUTABLE_DOCKER_TAG) $(DOCKER_IMAGE_PREFIX)brigade-$*:$(MUTABLE_DOCKER_TAG)
+	$(KANIKO_DOCKER_CMD) /kaniko/executor \
```

Comment:
Was there a reason the meta targets of `build` and `build-images` were removed?

Reply:
I found that they were never being used. As I hack away I use the `hack-` targets. Our CI/CD pipelines also never called those because they're very parallelized and thus use the more finely-grained targets.

But now that you've questioned it, perhaps these should remain for the benefit of anyone who just wants to manually kick off a build of the whole shootin' match.

I'm going to put them back.

Reply:
I put 'em back.